### ISSUE TITLE
glimmer package --> Error: Invalid Option: [packages/bundles]

### ISSUE THREAD

**Issue by mistergibson** [AndyObtiva/glimmer#7]
jpackage --type  --dest 'packages/bundles' --input 'dist' --main-class JarMain --main-jar 'gxg_studio_glimmer.jar' --name 'Gxg Studio Glimmer' --vendor 'Gxg Studio Glimmer' --icon 'package/windows/Gxg Studio Glimmer.ico'  --app-version "1.0.0"  --license-file LICENSE.txt  --copyright "Copyright (c) 2020 studio"   -name 'Studio' -title 'Studio' -Bmac.CFBundleName='Studio' -Bmac.CFBundleIdentifier='org.studio.application.Studio'   
WARNING: Using incubator modules: jdk.incubator.jpackage
Error: Invalid Option: [packages/bundles]

Am I missing a build .deb or other package/dep?

**Comment by AndyObtiva** [comment]
It says "jpackage" meaning it is not on the required JDK 8 (which has javapackager instead, fully supported by Glimmer). "jpackage" support is currently experimental not official. Make sure you are running the Java version (JDK 8u241 (1.8.0_241)) mentioned in the pre-requisites: https://github.com/AndyObtiva/glimmer-dsl-swt#pre-requisites 

One more thing, Linux is officially supported via Ruby gem packaging, not native .deb packaging, at the moment. If you are on Linux, scaffold a [custom shell gem](https://github.com/AndyObtiva/glimmer-dsl-swt#custom-shell-gem) instead of an "app", and it 
comes with a Linux binary executable shell (e.g. https://github.com/AndyObtiva/glimmer-cs-calculator comes with a `bin/calculator` to run it on Linux). It can also be packaged as a Ruby gem with the [Juwelier](https://github.com/flajann2/juwelier) (rake build) command, generating a gem file under `pkg` directory, which can be published (with `rake release`) just like glimmer-cs-calculator or just installed locally without publishing with `rake install`. I might have to polish it more and provide a Glimmer friendly command on Linux instead of just Juwelier, and probably improve the Glimmer documentation for Linux. This is the solution until then though (`glimmer scaffold:gem:customshell[app_name,namespace]` and then `rake build` inside the app once done, and finally after installing the gem, just run the executable shell).

What is your operating system? Are you using one operating system or multiple? Please provide in the future with any help inquiries.

**Comment by AndyObtiva** [comment]
OK, I just ran scaffolding in Linux, and noticed it does not launch the app in the end, and it runs standard packaging (like Mac and Windows) instead of gem packaging. I messed up on this and apologize for any inconvenience this may have caused.

The good news is that the app is scaffolded correctly otherwise and does run with `glimmer run`. Also, I could still package as a gem with the `rake build` command (but it only comes with the binary shell script in `custom shell gem` mode not in `app` mode.)
